portfolio/src/main/webapp/script.js

```diff
@@ -1,3 +1,19 @@
 function changeImage() {
   document.getElementById("switch").className = "index-image-2"; 
-}
\ No newline at end of file
+}
+
+function createListElement(text) {
```

Comment:
nit: add JSDoc to the functions in this file. 

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments